Team2-Signoff-LightingControl.md

Noah-Jones37 opened the pull request:
Requesting signoff for lighting control.

Noah-Jones37 committed:
Create Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
Not a buildable schematic.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Add files via upload

jax-commits committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
This is not how these items would actually be wired. The system constraints aren't sufficiently defined.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
No analysis of the photocell. This is apparently uncontrollable as it has no inputs. Also, you have no citations in the text for things that are claimed regarding plants. That is very necessary here.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
Complete constraints and design

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
This does not accomplish the result we discussed. According to this you need a way to control the distance of the light. further, you'll need more than one bulb.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
This needs to be entirely automatic. This is an automation project. 

For this to be signed off, this needs to be a very good design that accomplishes the goals of the system. Which is not at all met by a single bulb and a manual position system. 

Design something that you are proud of.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
Discussion in class

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
These huge blocks of text in the analysis section make it so nothing is clear. Clean this up and show the reader that the design is appropriate by explaining what you are checking, and how you know that it is appropriate. 

I don't see the box in any schematics. How do you know your box is big enough?

Your schematic is a direct copy of what I drew. However, I don't see that it actually matches the relay.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
Still has giant blocks of text for analysis.

JesseTNRoberts reviewed (commented):
Make sure that you have checked ever current and every voltage rating on every component. I want to see in the analysis that you have checked every component to ensure that it is appropriate to the application. D

As an example, don't simply say that 14 gauge is sufficient. Explain how you know and cite your source.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

In the power supply analysis:
Line 2 - breaker proof
Line 3 - relay proof
Line 4 - extension cord proof

JesseTNRoberts reviewed (commented):
Please include excerpts from each of the relay and circuit breaker data sheets showing the values that you have claimed.

Also, the enclosure will need a backplane to mount the components to. often this is included. However, this is not always the case and needs to be verified.

jax-commits committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
If the max switching current is 10 amps, and the circuit won't break until 15, the relay is inappropriate.

also review past comments.

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

JesseTNRoberts reviewed (commented):
Better. But we aren't going to pay 60 dollars for a relay. That is about 6 times the appropriate price.

Also, I see 3 extension cords being ordered but it seems we need two for the design?

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

Noah-Jones37 committed:
Update Lighting Control - Signoff.md

jax-commits committed:
Merging LightingControl
